### ISSUE TITLE
searchAttachmentsContainer

### ISSUE THREAD

**Issue by lnmunhoz** [envisioning/techdb#50]
### Specification

This container integrates searchContainer #48 with attachmentCard #49 and recentUpdates #34 to create a nice view end attachment management.

![searchattachmentscontainer](https://cloud.githubusercontent.com/assets/4376835/12306200/b29ceb0e-ba1f-11e5-8a94-69a7f9a51af0.png)

**Comment by arthursoares** [comment]
+1

**Comment by rafaelcorreiapoli** [comment]
Where should this be placed? attachmentsDashboard?

**Comment by rafaelcorreiapoli** [comment]
@arthursoares

**Comment by rafaelcorreiapoli** [comment]
@arthursoares

**Comment by arthursoares** [comment]
@rafaelcorreiapoli yes! Actually that seems like a good default behavior for every search box in the dashboards.

But I think it is some steps ahead from where we are.
